backend/src/main/java/ba/unsa/etf/si/bbqms/admin_service/implementation/DefaultStationService.java

```diff
@@ -128,4 +128,18 @@ public Set<TellerStation> getAllOfferingService(final Service service) {
     public Optional<TellerStation> findById(final long serviceId) {
         return this.tellerStationRepository.findById(serviceId);
     }
+
+    @Override
+    public Set<Service> getServicesForTellerStation(long stationId) {
+        return this.tellerStationRepository.findById(stationId)
+                .map(TellerStation::getServices)
+                .orElseThrow(EntityNotFoundException::new);
+    }
+
+    @Override
+    public long getBranchIdForTellerStation(final long stationId) {
+        return this.tellerStationRepository.findById(stationId)
+                .map(tellerStation -> tellerStation.getBranch().getId())
+                .orElseThrow(EntityNotFoundException::new);
+    }
```

Comment:
This method will not be necessary (see below why), and even if it was, we should always return the whole object and not just ID.